compiler/noirc_evaluator/src/ssa/ir/function.rs

```diff
@@ -217,6 +218,17 @@ impl Function {
             })
             .sum()
     }
+
+    /// Iterate over the numeric constants in the function.
+    pub fn constants(&self) -> impl Iterator<Item = (&FieldElement, &NumericType)> {
+        self.dfg.values_iter().filter_map(|(_, value)| {
+            if let Value::NumericConstant { constant, typ } = value {
```

Comment:
This looks like it would be missing global constants?

Reply:
Good point! Chained them in.